zcash_client_sqlite/src/wallet.rs

```diff
@@ -206,7 +206,7 @@ impl Account {
     ) -> Result<(UnifiedAddress, DiversifierIndex), AddressGenerationError> {
```

Comment:
This method can be public.

Reply:
In general we would like to avoid encouraging users to use the `default_address` method. In any case, this is not a change that should be made in this PR.